### PR TITLE
Ruby Logging Docs

### PR DESCRIPTION
**Pull request by orangewolf**
Found this info hiding in the Discussions tab on Github for Lograge and had to cobble it together via StackOverflow for the default logger. Felt like it would be good to include in the official documentation.

### CODE REVIEW

**Comment by svrnm** [comment]
Hey @orangewolf, thank you for raising this PR. Unfortunately, we can not accept it: the mechanism that is described here is a work around to inject trace context into logs with the [Logs API & SDK](https://opentelemetry.io/docs/specs/otel/logs/) being not available (I leave it to the @open-telemetry/ruby-approvers  & @open-telemetry/ruby-maintainers to comment on their status with logs). The language SIGs are working hard on providing those and when they are available this way of injecting trace & span context into your logs is discouraged. And when they are available, we (the docs team) are going to make sure they are available for end-users. Until then we will also not provide official documentation on this work around, as it my send a wrong signal to end users, i.e. this may lead them to assume that this is how logging is *supposed* to be done with OpenTelemetry.

Apologies that I have to reject your PR, as I appreciate you taking the time to write down your learnings and your willingness to share it with the community.

**Review by chalin — COMMENTED** [review]
Drive-by comments, while awaiting Ruby SIG comments about if / how best to include this content. See inline suggestion.

Also, unindent the Ruby code in the code blocks.

File: content/en/docs/instrumentation/ruby/logging.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+You can added Trace and Span IDs to logging. This will allow easy connection between a trace and the corrisponding logs. This feature is frequently supported by viewing tools such as [Tempo](https://grafana.com/oss/tempo/) or [Signoz](https://signoz.io). How you configure your logging will depend on the logging library your Ruby application uses. 
+
+### Rails Default Logger
+
+By default Rails uses the TaggedLogger. Span and Trace IDs can be added like this:
+
+```ruby
+      config.log_tags = [ 
+        :trace_id,
+        proc { span = OpenTelemetry::Trace.current_span rescue nil; span&.context&.hex_trace_id },
```
[the remark this answers is not in the document]
Comment (robertlaurin) [reply content/en/docs/instrumentation/ruby/logging.md:18]: I need to think through the different failure modes here, but if the valid check is what determines whether or not we write the trace id to the logs, we might actually instead want to check `.recording?`. A valid trace id may not actually be recording, and in that case the logger would be emitting trace ids that cannot be found.

**Comment by cartermp** [comment]
Closing due to inactivity

**Pull request closed without merging**